src/Common/src/Interop/Windows/Interop.Libraries.cs

```diff
@@ -23,6 +23,7 @@ internal static partial class Libraries
         internal const string PerfCounter = "perfcounter.dll";
         internal const string RoBuffer = "api-ms-win-core-winrt-robuffer-l1-1-0.dll";
         internal const string Secur32 = "secur32.dll";
+        internal const string SChannel = "schannel.dll";
```

Comment:
Pls investigate if you can use 'secur32.dll' for this p/invoke.

Reply:
Seems like only in schannel.dll, getting EntryPointNotFoundException with secur32